src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.html

```diff
@@ -7,6 +7,7 @@
         <ul ui-sortable="sortableOptions" ng-model="images" class="umb-sortable-thumbnails">
             <li data-element="sortable-thumbnail-{{$index}}" class="umb-sortable-thumbnails__wrapper" ng-repeat="image in images track by $index">
 
+                <span class="label trashed" ng-if="image.trashed">trashed</span>

```

Comment:
Would be nice if this label is already localized

Reply:
@dawoe agree but sorry i don't know how to do that - I haven't contributed to core since 4.7ish! If you can tell me what I need to put to get that label localized i'll drop it in my branch which will update the PR

Reply:
hi @agrath 

You can find the files with translations here : https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web.UI/umbraco/config/. Probably best to put in the mediapicker area.

And in your angular view you can use the localize directive. https://our.umbraco.com/apidocs/ui/#/api/umbraco.directives.directive:localize